pkg/services/store/entity_events.go

```diff
@@ -90,36 +88,6 @@ type entityEventService struct {
 	eventHandlers []EventHandler
 }
 
-func (e *entityEventService) SaveEvent(ctx context.Context, cmd SaveEventCmd) error {
-	entityEvent := &EntityEvent{
-		EventType: cmd.EventType,
-		EntityId:  cmd.EntityId,
-		Created:   time.Now().Unix(),
-	}
-	err := e.sql.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
-		_, err := sess.Insert(entityEvent)
-		return err
-	})
-	if err != nil {
-		return err
-	}
-	return e.broadcastEvent(ctx, entityEvent)
-}
-
-func (e *entityEventService) broadcastEvent(ctx context.Context, event *EntityEvent) error {
```

Comment:
I am not following completely the new search. The code you removed actually do insert event into entity_event table, and then broadcast event after. In the new added code, there is no longer broadcast event?

Reply:
Hello, yep - we are not broadcasting it anymore. This was a way to achieve consistent search reads after changing dashboard. But it did not work for HA case. So at this moment we switched to https://github.com/grafana/grafana/pull/50152 approach.

I'll update PR description - had to mention that this pr also remove broadcasting logic.